src/value.cc

```diff
@@ -35,6 +36,22 @@ static void HashPointerToGIArgument (GIArgument *arg, GITypeInfo *type_info);
 
 static bool IsUint8Array (GITypeInfo *type_info);
 
+static int64_t V8ToInt64 (Local<Value> value) {
+    if (value->IsBigInt())
+        return value.As<v8::BigInt>()->Int64Value();
+    return Nan::To<int64_t>(value).ToChecked();
+}
+
+static int64_t V8ToUint64Strict (Local<Value> value) {
+    return Local<BigInt>::Cast(value)->Uint64Value();
+}
+
+static int64_t V8ToInt32 (Local<Value> value) {
+    if (value->IsBigInt())
+        return value.As<v8::BigInt>()->Int64Value();
+    return Nan::To<int32_t>(value).ToChecked();
+}
```

Comment:
This one feels a bit weird, with the name being `ToInt32` but it returns an `int64_t`. Maybe it should return an `int32_t`?